docs/workstation_release_management.rst

```diff
@@ -36,11 +36,9 @@ In addition, we have the following (Debian) metapackages, which are
 stored in the
 `securedrop-debian-packaging <https://github.com/freedomofpress/securedrop-debian-packaging>`__
 repository: \*
-```securedrop-workstation-config`` <https://github.com/freedomofpress/securedrop-debian-packaging/tree/master/securedrop-workstation-config/debian>`__
+```securedrop-workstation-config`` <https://github.com/freedomofpress/securedrop-debian-packaging/tree/main/securedrop-workstation-config/debian>`__
 \*
-```securedrop-workstation-grsec`` <https://github.com/freedomofpress/securedrop-debian-packaging/tree/master/securedrop-workstation-grsec>`__
-\*
-```securedrop-workstation-viewer`` <https://github.com/freedomofpress/securedrop-debian-packaging/tree/master/securedrop-workstation-viewer>`__
+```securedrop-workstation-viewer`` <https://github.com/freedomofpress/securedrop-debian-packaging/tree/main/securedrop-workstation-viewer>`__
```

Comment:
Note that the formatting of this paragraph is broken (before and after your changes), I'll open a separate PR to fix.